.travis.yml

```diff
@@ -9,9 +9,9 @@ before_install:
   - cask install
 
 env:
-  - TAGS="" EVM_EMACS=emacs-24.5-travis
```

Comment:
Why is this neccesary? Did I accidentally use some functionality that isn't available in emacs-24.5?

Reply:
> Why is this neccesary? Did I accidentally use some functionality that isn't available in emacs-24.5?

I can add back Emacs 24.5, but I'd rather not. Emacs 24.5 is from 2015 and many things changed since then, I don't think we should aim to actively support it.

Reply:
Do you think we should support at least `25.1`? I think this isn't that hard, and already gives us some niceties like `user-error`.

Reply:
What about the supporting the latest stable release from the 25.x branch?

Reply:
Which one would that be?

Reply:
That would be 25.3.

Reply:
Sure, 25.3 is fine for now. I can look into 25.1 compatibility later.

Reply:
> Sure, 25.3 is fine for now. I can look into 25.1 compatibility later.

I believe that `js2-refactor` **is compatible** with Emacs 25.1. What I meant is that by testing against 25.3 and newer, we would only actively support `>= 25.3`.

Reply:
I actually meant actively supporting 25.1. However, 25.3 is good enough for now; I think we should do the modularization + autoload refactor first. Can you change the minimum Emacs version in Travis CI to 25.3 so that this can be merged?